barretenberg/cpp/src/barretenberg/honk/sumcheck/sumcheck.test.cpp

```diff
@@ -107,11 +107,11 @@ TEST_F(SumcheckTests, PolynomialNormalization)
 
     auto sumcheck = SumcheckProver<Flavor>(multivariate_n, transcript);
 
-    auto [multivariate_challenge, evaluations] = sumcheck.prove(full_polynomials, {});
+    auto output = sumcheck.prove(full_polynomials, {});
 
-    FF u_0 = multivariate_challenge[0];
-    FF u_1 = multivariate_challenge[1];
-    FF u_2 = multivariate_challenge[2];
+    FF u_0 = output.challenge_point[0];
```

Comment:
i think `point` is not an ideal name cus it makes me think of elliptic curve points and this is a scalar.. maybe call the vector `challenges`

Reply:
renamed to "challenge" (singular since it's an vector of field elements interpreted as a multivariate challenge)